packages/salesforcedx-apex-debugger/src/messages/i18n.ts

```diff
@@ -30,5 +30,5 @@ export const messages = {
   streaming_handshake_timeout_text:
     'Streaming API handshake failed due to a socket timeout. Check your connection and try again.',
   hotswap_warn_text:
-    'Apex classes & triggers should not be modified during an active Apex Debugger session.'
+    "You can't modify Apex classes or triggers during an Apex Debugger session. Save your changes after you’re done debugging."
```

Comment:
There's one curly apostrophe and one straight apostrophe in this message. I'd prefer that we use curly apostrophes, but if we need to use straight ones we should do it consistently.

Reply:
@DatGuyJonathan ^^^

Reply:
Good catch! There's [this](https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-vscode-core/src/messages/i18n.ts#L46) other usage of straight apostrophe, so I'll stick with that.